phpBB/phpbb/console/command/db/console_migrator_output_handler.php

```diff
@@ -54,11 +54,11 @@ public function write($message, $verbosity)
 		{
 			$translated_message = call_user_func_array(array($this->user, 'lang'), $message);
 
-			if ($verbosity === migrator_output_handler_interface::VERBOSITY_NORMAL)
+			if ($verbosity === OutputInterface::VERBOSITY_NORMAL)
```

Comment:
that or just map the ones in the console migrator output handler, or whatever. it makes no difference, and it is mostly just a question of style, but i don't see why it would be reasonable to include a class just for its constants. also the symfony console is designed for the console, while the output handler is not, and therefore their similarity should be treated as a coincidence imho (meaning only be used in the console output handler).